### ISSUE TITLE
Update to 3.1.0-beta5 - Invalid JSON data on Craft migrations

### ISSUE THREAD

**Issue by smcyr** [craftcms/cms#3560]
### Description

I tried to update to Craft 3.1.0-beta5 but got this error:
```
One of Craft CMS’s migrations failed.
yii\base\InvalidArgumentException: Invalid JSON data.
```

The error is in the file `vendor/craftcms/cms/src/services/ProjectConfig.php`, method `_getStoredConfigMap()` on line 885. `Craft::$app->getInfo()->configMap` return this array:
```
[
dateModified = "D:\projects\craft3\config\project.yaml"
email = "D:\projects\craft3\config\project.yaml"
fieldGroups = "D:\projects\craft3\config\project.yaml"
fields = "D:\projects\craft3\config\project.yaml"
globalSets = "D:\projects\craft3\config\project.yaml"
plugins = "D:\projects\craft3\config\project.yaml"
sections = "D:\projects\craft3\config\project.yaml"
siteGroups = "D:\projects\craft3\config\project.yaml"
sites = "D:\projects\craft3\config\project.yaml"
system = "D:\projects\craft3\config\project.yaml"
users = "D:\projects\craft3\config\project.yaml"
volumes = "D:\projects\craft3\config\project.yaml"
]
```
But `Json::decode` doesn't expect an array.

### Additional info

- Apache on Windows
- Craft version: 3.1.0-beta5
- PHP version: 7.2

**Comment by smcyr** [comment]
I think that the problem is this migration: `m181213_102500_remove_absolue_paths_from_configmap`. There is `Json::decode()` on the config map but no `Json::encode()` before saving.
Also, the `Craft::alias($filePath)` doesn't seem to do anything in my case.

**Comment by smcyr** [comment]
It's working if I modify line 29 of `m181213_102500_remove_absolue_paths_from_configmap`:
```
$info->configMap = Json::encode($configMap);
```

**Comment by brandonkelly** [comment]
Just released 3.1.0-beta.5.1 with a fix for this.
